src/Modulo2.jl

```diff
@@ -663,6 +679,16 @@ function copyto!(a::ZZ2Array{N}, bc::Broadcasted{ZZ2ArrayStyle{N}, <:Any, typeof
     a
 end
 
+#
+# Utility functions
+#
+
+count_ones(a::ZZ2) = ifelse(a.m, 1, 0)
+count_zero(a::ZZ2) = ifelse(a.m, 0, 1)
```

Comment:
```suggestion
count_ones(a::ZZ2) = Int(isodd(a))
count_zeros(a::ZZ2) = Int(iseven(a))
```
This way we use existing functions.